app/docker/views/services/edit/service.html

```diff
@@ -78,63 +78,76 @@
               <td ng-if="webhookURL">
                 <span class="text-muted">{{ webhookURL | truncatelr }}</span>
                 <button type="button" class="btn btn-sm btn-primary btn-sm space-left" ng-if="webhookURL" ng-click="copyWebhook()">
-                  <span><i class="fa fa-copy space-right" aria-hidden="true"></i>Copy link</span>
+                  <span>
```

Comment:
in EE this span was removed (to be expected as it is not needed). please remove it here too

Reply:
updated